contracts/CO30_Emerson_RedSkyAtNight.Tanoa/scripts/crateFall.sqf

```diff
@@ -0,0 +1,54 @@
+// enable simulation for crate 
+["FARE_enableSimulation", [crate], crate] call CBA_fnc_targetEvent;
+
+crate setPosASL (getPosASL crate);
+
+// play crate sound 
+playSound3D [
+    getMissionPath "sounds\crateFall.ogg", 
+    crate, 
+    false, 
+    getPosASL crate, 
+    5, 
+    1, 
+    0
+];
+
+// enable all simulation for dude
+["FARE_enableSimulation", [crateDude], crateDude] call CBA_fnc_targetEvent;
+
+// enable all simulation for radio speaker
+["FARE_enableSimulation", [crateWorker], crateWorker] call CBA_fnc_targetEvent;
+
+// need to hurt the guy ourselves after crate falls and then play radio message
+[
+    {
+        // enable damage for dude 
+        ["FARE_allowDamage", [crateDude], crateDude] call CBA_fnc_targetEvent;
+
+        // hurt him a lot!
+        [crateDude, 1, "Body", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 0.4, "RightLeg", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 1, "LeftLeg", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 0.4, "RightArm", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 1, "LeftArm", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 0.4, "Head", "falling"] call ace_medical_fnc_addDamageToUnit;
+
+        [crateDude, 0.5] call ace_medical_fnc_adjustPainLevel;
```

Comment:
These only apply to a local unit, this whole thing script should be an event that tis called via targetEvent in the trigger